src/_gettsim/parameters/sozialv_beitr.yaml

```diff
@@ -165,7 +165,7 @@ beitr_satz:
     ges_rentenv: 0.0995
   2008-01-01:
     ges_krankenv:
-      allgemein: 0.14
+      mean_allgemein: 0.14
```

Comment:
We made the differentiation between `allgemein` and `mean_allgemein` on purpose. The latter refers to a period in which health insurance contribution rates differed over health insurance providers and we report the mean of them.

Let's discuss the best interface here.